recipes/bokeh/run_test.py

```diff
@@ -0,0 +1,8 @@
+import sys
+import bokeh
+
+if sys.platform != 'win32':
+    bokeh.test()
```

Comment:
don't know, this is inherited from anaconda-recipes, as the missing LICENSE file I forgot to add should indicate.